core/src/thread_parker/linux.rs

```diff
@@ -5,19 +5,26 @@
 // http://opensource.org/licenses/MIT>, at your option. This file may not be
 // copied, modified, or distributed except according to those terms.
 
+// REVIEW: this module needs a doc comment going into more depth about how it's
+// implementing the `ThreadParker` protocol.
+
 use super::libstd::{thread, time::Instant};
 use core::{
     ptr,
     sync::atomic::{AtomicI32, Ordering},
 };
 use libc;
 
+// REVIEW: these constants should be added to libc, verified, and used from
+// there
```

Comment:
These constants aren't actually provided by libc (the glibc/musl headers don't have them). They are part of the kernel headers, which may or may not be in the scope of libc.

Reply:
I'm adding them over at https://github.com/rust-lang/libc/pull/1336